resume_matcher.py

```diff
@@ -41,26 +88,32 @@ def choose_api():
     
     print(colored(f"\nSelected API: {chosen_api.capitalize()}", "green", attrs=["bold"]))
 
-def talk_to_ai(prompt, max_tokens=1000, image_data=None, client=None):
+def talk_to_ai(prompt,
```

Comment:
Do you want to take it 1-step further and plug https://docs.ell.so in? It can do both: OpenAI and Anthropic, including images. 

```
import ell

@ell.simple(model="gpt-4o-mini")
def hello(world: str):
    """You are a helpful assistant""" # System prompt
    name = world.capitalize()
    return f"Say hello to {name}!" # User prompt

hello("sam altman") # just a str, "Hello Sam Altman! ..."
```

Reply:
++